sdk/textanalytics/ai-text-analytics/test/collections.spec.ts

```diff
@@ -200,7 +200,9 @@ describe("RecognizeCategorizedEntitiesResultArray", () => {
             {
               text: "Microsoft",
               category: "Organization",
-              confidenceScore: 0.9989
+              confidenceScore: 0.9989,
```

Comment:
confidenceScore will changes based on vary model service used so it is better not validate the exact number. But length and offset might be worth to validate since it with be fixed.

Reply:
this tests operations on collections and does not call the service.